src/sentry/api/endpoints/organization_transaction_anomaly_detection.py

```diff
@@ -0,0 +1,92 @@
+from datetime import datetime
+
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.bases import OrganizationEventsEndpointBase
+from sentry.api.utils import get_date_range_from_params
+from sentry.search.events.builder import QueryBuilder
+from sentry.snuba.dataset import Dataset
+
+
+class OrganizationTransactionAnomalyDetectionEndpoint(OrganizationEventsEndpointBase):
```

Comment:
Is that a good class name?
Not sure if I got the right base class.

Reply:
class name is good enough, don't forget to add this to api/urls.py and add some unit tests too

Reply:
Lemme get to unit testing.